### DISCUSSION TITLE
Prestige template is not available

### DISCUSSION THREAD

**Discussion by miroslavgacha** [primefaces/community#1600] [PrimeNG Templates] [answered]
Hi team, we bought the Prestige template in June 2023 and it is not available now in the store and on link https://www.primefaces.org/prestige-ng

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Prestige is one of the templates that has been retired because it's outdated. It's not available to buy anymore but we'll provide a manual update guide for the outdated templates so the users can use them in recent Angular versions.
